crates/matrix-sdk-crypto/Cargo.toml

```diff
@@ -47,23 +47,8 @@ sha2 = "0.10.2"
 thiserror = "1.0.30"
 tracing = "0.1.34"
 zeroize = { version = "1.3.0", features = ["zeroize_derive"] }
-
```

Comment:
Well someone could also take the `matrix-sdk` crate and use it in a pure WASM project using [Yew](https://github.com/yewstack/yew), no? As far as I understand, such a project will need to enable the `js` feature of vodozemac somehow.

Seems like we don't use quite the right thing for the matrix-sdk crate, we enable the `getrandom` `js` feature, but we might be using different versions of getrandom since vodozemac is stuck on an older `rand` crate: 
https://github.com/matrix-org/matrix-rust-sdk/blob/2ff6497604c33ab0acec38753cd0fe33106acc31/crates/matrix-sdk/Cargo.toml#L146

Reply:
Restored.